sql/core/src/main/scala/org/apache/spark/sql/execution/SparkStrategies.scala

```diff
@@ -235,8 +235,13 @@ abstract class SparkStrategies extends QueryPlanner[SparkPlan] {
           .getOrElse(createJoinWithoutHint())
 
       case j @ ExtractSingleColumnNullAwareAntiJoin(leftKeys, rightKeys) =>
```

Comment:
We should also update the doc of `spark.sql.optimizeNullAwareAntiJoin`. It is not only `BroadcastHashJoinExec` now.

Reply:
OK.